aptos-move/framework/aptos-framework/sources/Account.move

```diff
@@ -343,18 +343,70 @@ module AptosFramework::Account {
     }
 
     ///////////////////////////////////////////////////////////////////////////
-    /// Basic account creation method.
+    /// Basic account creation methods.
     ///////////////////////////////////////////////////////////////////////////
 
     public(script) fun create_account(auth_key: address) {
-        let (signer, _) = create_account_internal(auth_key);
+        let signer = create_account_internal(auth_key);
         Coin::register<TestCoin>(&signer);
     }
 
+    public fun create_module_account(
```

Comment:
should this be called create_shared_account?  module_account makes me think of the program account in ETH.

Reply:
That's exactly the goal :P